openfecwebapp/api_caller.py

```diff
@@ -45,6 +45,40 @@ def load_search_results(query, query_type='candidates'):
     return results['results'] if len(results) else []
 
 
+def load_legal_search_results(query, query_type='all', limit=20):
+    filters = {}
+
+    def _transform_advisory_opinion(advisory_opinion):
+        source = advisory_opinion['_source']
+        return {
+            'id': source['AO_Id'],
+            'no' : source['AO_No'],
+            'name': source['AO_name'],
+            'summary': source['AO_Summary'],
+            'tags': source['AO_tags'],
+            'description': source['description'],
+            'doc_id': source['doc_id'],
+            'text': source['text'],
+        }
+
+
+    def _transform(data):
```

Comment:
Just curious--what about applying some or all of these transforms on the api side? Should the api serve idiosyncratically named elastic-style fields like `_type` and `_source`?